src/mds/Server.cc

```diff
@@ -1291,6 +1204,100 @@ void Server::respond_to_request(MDRequestRef& mdr, int r)
   }
 }
 
+// statistics mds req op number and latency 
+void Server::mds_op_count(__u32 op, utime_t lat)
```

Comment:
how about rename it to perf_gather_op_latency

Reply:
ok